src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.html

```diff
@@ -67,4 +67,10 @@
   <ng-container i18n
                 *ngIf="showSubmit">Are you sure you want to continue?</ng-container>
 </ng-template>
+
+<ng-template #orchTmpl>
+  <span i18n
+        i18n-ngbTooltip
+        ngbTooltip="Data will be available only if Orchestrator is available.">Not Avail.</span>
```

Comment:
IMO we should avoid using abbreviations here. Can we change it to `Unavailable` or `No Data`.